### PR TITLE
Graphical recording buttons

### PR DESCRIPTION
**Pull request by nocommentcode**
Added the `ActionBar` component to graphical recording mode

- Added the button icons to the assets folder
- Added a style.ts file for `GraphicalRecordingMode`
- Added the `ActionBar` component to graphical recording mode
- Wrapped the `ChessBoard` and `ActionBar` components in flex box
- Linked the flip board button as it is easy to do

image

### CODE REVIEW

File: packages/TorneloScoresheet/src/pages/GraphicalRecording/GraphicalRecording.tsx [hunk 1]
```diff
@@ -21,14 +91,18 @@ const GraphicalRecording: React.FC = () => {
...
-        <View>
-          <ChessBoard
-            positions={graphicalRecordingMode.board}
-            onMove={onMove}
-            flipBoard={
-              graphicalRecordingMode.currentPlayer === PlayerColour.Black
-            }
-          />
+        <View style={styles.mainContainer}>
+          <View style={{ height: 100 }}>
```
Comment (gestureleft) [inline packages/TorneloScoresheet/src/pages/GraphicalRecording/GraphicalRecording.tsx:95]: Inline styling here as well - let's put this in `style.ts`

**Review by gestureleft — CHANGES_REQUESTED** [review]
Nice! One small change

File: packages/TorneloScoresheet/src/pages/GraphicalRecording/GraphicalRecording.tsx (hunk 1 above)
Comment (nocommentcode) [reply packages/TorneloScoresheet/src/pages/GraphicalRecording/GraphicalRecording.tsx:95]: this is just a placeholder though, will be replaces by the player cards once we implement it

Comment (gestureleft) [reply packages/TorneloScoresheet/src/pages/GraphicalRecording/GraphicalRecording.tsx:95]: True! My bad

Comment (gestureleft) [reply packages/TorneloScoresheet/src/pages/GraphicalRecording/GraphicalRecording.tsx:95]: Should probably have a TODO/FIXME there but doesn't matter - I'll approve now

**Review by gestureleft — APPROVED** [review]
🔥

**Pull request merged**